apps/sveltekit-example-app/src/routes/api/[...slug]/+server.ts

```diff
@@ -0,0 +1,5 @@
+import type { RequestHandler } from '@sveltejs/kit'
+
+import api from '@packages/api'
+
+export const fallback: RequestHandler = ({ request }) => api.fetch(request)
```

Comment:
Since we want everything coming thru this route to go to Hono,
trying out the fallback approach rather than specifying each
HTTP method.